.eslintrc.es6.json

```diff
@@ -0,0 +1,42 @@
+{
+    "extends": "ovh",
+    "globals": {
+        "window": false,
```

Comment:
It seems there is a mistake anywhere because you don't have to do that I think it's already included in ovh-config